doc/blog/201809-UnoUnderTheHoodXamlGeneration.md

```diff
@@ -0,0 +1,84 @@
+# Talkin' 'bout my generation: How the Uno Platform generates code, part 1
```

Comment:
do we need "the" ? How Uno Platform generates code

Reply:
no it sounds weird